### PR TITLE
feat: add teams page

### PR DESCRIPTION
**Pull request by gokul-00**
- Implemented teams&captain page
- content structure added in assets directory

### CODE REVIEW

File: sports-nitt/package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@mui/material": "^5.0.2",
```
Comment (ahmed-28) [inline sports-nitt/package.json]: why did you remove mui  dependency?

File: sports-nitt/src/assets/content/teams.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+[
+	{
+		"teamName": "Football",
+		"captainName": "Lionel Messi",
```
Comment (ahmed-28) [inline sports-nitt/src/assets/content/teams.json]: There can be multiple captains (boy and girl)?

File: sports-nitt/src/pages/teamsPage/Team.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		<div
+			class="banner-img"
+			style={{
+				backgroundImage: `url(${data.teamImgUrl})`,
+				backgroundRepeat: "no-repeat",
+			}}
+		>
+			<h4 class="team">{data.teamName}</h4>
+		</div>
+		<img src={data.capImgUrl} alt="captain-img" class="profile-img" />
```
Comment (ahmed-28) [inline sports-nitt/src/pages/teamsPage/Team.jsx]: dont assume one captain..let it be one or two 
so make changes accordinly..
if it is one captain then position it in center
If two then make two img equally spaced..

**Review by ahmed-28 — COMMENTED** [review]
Also, there was some error regarding required modules, Please look into it
I manually added somthing called colerrete and also eslint 
I am not sure why this occured anyways i'll check again.

File: sports-nitt/package.json (hunk 1 above)
Comment (gokul-00) [reply sports-nitt/package.json]: but its there in local repo. anyway i reinstalled it
